libyui-qt-pkg/src/YQPkgHistoryDialog.cc

```diff
@@ -29,277 +29,357 @@
 #include <yui/qt/YQi18n.h>
 #include <yui/qt/YQUI.h>
 #include <yui/qt/utf8.h>
+#include <yui/qt/YQSignalBlocker.h>
 
-#include <zypp/parser/HistoryLogReader.h>
 #include <zypp/Date.h>
 #include <zypp/Edition.h>
 #include <boost/ref.hpp>
 
 #include <QApplication>
-#include <QDesktopWidget>
+#include <QBoxLayout>
+#include <QEventLoop>
 #include <QHBoxLayout>
 #include <QLabel>
+#include <QList>
+#include <QMessageBox>
 #include <QPushButton>
 #include <QSplitter>
-#include <QStyle>
-#include <QList>
-#include <QBoxLayout>
 #include <QTreeWidget>
-#include <QMessageBox>
 
 #include "YQPkgHistoryDialog.h"
 #include "YQPkgList.h"
 #include "YQIconPool.h"
 #include "QY2LayoutUtils.h"
 
 
-#define SPACING			2	// between subwidgets
-#define MARGIN			4	// around the widget
+#define SPACING         4	// between subwidgets
```

Comment:
> Now using Qt-standard margins (9 around the dialog) and spacings (4 between widgets).

Is that defined somewhere in Qt? Could we use some Qt constant here instead of these "magic" numbers?

Reply:
I don't know any constants for this. It's largely dependent on what Qt widget style is used.